lib/ansible/module_utils/alicloud_ecs.py

```diff
@@ -0,0 +1,176 @@
+#
```

Comment:
@xiaozhu36 if there are other files in module_utils without a bsd license, that is a bug. We will not accept new contributions to module_utils that are not bsd licensed. The explanation can be found here:

https://groups.google.com/forum/#!topic/ansible-devel/zLyEYxRSvaw